basic/declare.s

```diff
@@ -184,3 +184,6 @@ usrpok	.res 3           ;$0310 user function dispatch
     exec_flag: .res 1
     exec_addr: .res 2
     exec_bank: .res 1
+
+    lp_dopause: .res 1
+    lp_screenpause: .res 1
```

Comment:
and just a minor whitespace nitpick, a newline at the end of file is preferred.  Some editors have the option to always add one if you edit a file.

In VSCode/VSCodium, that option is under Settings (Ctrl+Comma), under the User tab, Text Editor, Files, Insert Final Newline